package.json

```diff
@@ -46,13 +48,15 @@
     "eslint-plugin-simple-import-sort": "5.0.2",
     "husky": "4.2.3",
     "lint-staged": "10.0.9",
+    "lodash": "^4.17.15",
     "ndarray": "1.0.19",
     "npm-run-all": "4.1.5",
-    "numeral": "2.0.6",
+    "numeral": "^2.0.6",
     "parcel": "1.12.4",
     "prettier": "2.0.2",
-    "react": "16.13.1",
-    "react-dom": "16.13.1",
+    "query-string": "^6.12.0",
```

Comment:
Note: I've heard comparisons of the `query-string` and `qs` packages before. I think the last time I checked, I ended up using `qs` because it supports nesting but `query-string` doesn't? For example, I think you can pass `{a: {b: 'c'}}` to `qs` but not `query-string`.

Anyway, not a big deal if you can't look into it, maybe we don't need that functionality. And we can always swap it out later pretty easily.

Reply:
Sounds good - will keep as query-string for now, but when you need nested query strings, please feel free to change this! 😄